.gitignore

```diff
@@ -1,4 +1,5 @@
+Manifest.toml
 *.jl.*.cov
 *.jl.cov
 *.jl.mem
-/Manifest.toml
+*.las
```

Comment:
Please remove this `*.las` from the ignore. We wont' have las files in this repository.

Reply:
I already suggested this change in a previous review BTW.

Reply:
I added it as I was working keeping *.las file in the same folder. Will remove as soon as we are ready to push final repo!